app/views/idv/shared/_document_capture.html.erb

```diff
@@ -36,7 +36,9 @@
       in_person_outage_message_enabled: IdentityConfig.store.in_person_outage_message_enabled,
       in_person_outage_expected_update_date: IdentityConfig.store.in_person_outage_expected_update_date,
       us_states_territories: us_states_territories,
-      doc_auth_selfie_capture: IdentityConfig.store.doc_auth_selfie_capture,
+      doc_auth_selfie_capture: {
+        enabled: IdentityConfig.store.doc_auth_selfie_capture_enabled,
+      },
```

Comment:
It'd also be nice to update this in the JavaScript code as well. [Currently](https://github.com/18F/identity-idp/blob/e13d3d56650030a6163a8ce545756067da8f9e42/app/javascript/packs/document-capture.tsx#L56-L58) we bypass type-checking on this data attribute and perform a `JSON.parse` without error checking. I feel like we could get rid of that function altogether if we passed through the individual value.

```diff
diff --git a/app/javascript/packs/document-capture.tsx b/app/javascript/packs/document-capture.tsx
index 1a27313ef..d5df599ec 100644
--- a/app/javascript/packs/document-capture.tsx
+++ b/app/javascript/packs/document-capture.tsx
@@ -31,2 +31,3 @@ interface AppRootData {
   acuantVersion: string;
+  docAuthSelfieCaptureEnabled: 'true' | 'false';
   flowPath: FlowPath;
@@ -54,8 +55,2 @@ function getServiceProvider() {
 
-function getSelfieCaptureEnabled() {
-  const { docAuthSelfieCapture } = appRoot.dataset;
-  const docAuthSelfieCaptureObject = docAuthSelfieCapture ? JSON.parse(docAuthSelfieCapture) : {};
-  return !!docAuthSelfieCaptureObject?.enabled;
-}
-
 function getMetaContent(name): string | null {
@@ -95,2 +90,3 @@ const {
   acuantVersion,
+  docAuthSelfieCaptureEnabled,
   flowPath,
@@ -112,2 +108,4 @@ const {
 
+const selfieCaptureEnabled = docAuthSelfieCaptureEnabled === 'true';
+
 let parsedUsStatesTerritories = [];
@@ -144,3 +142,3 @@ const App = composeComponents(
       passiveLivenessOpenCVSrc: acuantVersion && `/acuant/${acuantVersion}/opencv.min.js`,
-      passiveLivenessSrc: getSelfieCaptureEnabled()
+      passiveLivenessSrc: selfieCaptureEnabled
         ? acuantVersion && `/acuant/${acuantVersion}/AcuantPassiveLiveness.min.js`
@@ -194,3 +192,3 @@ const App = composeComponents(
         exitQuestionSectionEnabled: String(uiExitQuestionSectionEnabled) === 'true',
-        selfieCaptureEnabled: getSelfieCaptureEnabled(),
+        selfieCaptureEnabled,
       },
diff --git a/app/views/idv/shared/_document_capture.html.erb b/app/views/idv/shared/_document_capture.html.erb
index d82035650..a33b0a70c 100644
--- a/app/views/idv/shared/_document_capture.html.erb
+++ b/app/views/idv/shared/_document_capture.html.erb
@@ -38,5 +38,3 @@
       us_states_territories: us_states_territories,
-      doc_auth_selfie_capture: {
-        enabled: IdentityConfig.store.doc_auth_selfie_capture_enabled,
-      },
+      doc_auth_selfie_capture_enabled: IdentityConfig.store.doc_auth_selfie_capture_enabled,
       skip_doc_auth: skip_doc_auth,
```

Reply:
I'll tag in @charleyf here since updating that portion of this seems firmly within the Timnit portion of this effort